README.adoc

```diff
@@ -162,16 +162,16 @@ include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClien
 ----
 This class is a client for the `system` service.
 
-In this class there is a method called `getPropertiesHelper()`. This method makes a request to the
-`system` service, and checks for the response. If a `Status.OK`` response which is corresponding to the
+A method called `getPropertiesHelper()` is included in this class. This method makes a request to the
```

Comment:
How about this to fix the passive voice? "There is a method called `getPropertiesHelper()` in this class.

Reply:
Unfortunately, "There is" is an expletive construction, which we try to avoid. What about "This class includes the `getPropertiesHelper()` method."?